feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html

```diff
@@ -1,3 +1,12 @@
+<!-- TO REMOVE -->
+<cx-spinner-button
+  (clikEvent)="test()"
+  [disabled]="true"
+  [text]="'Test'"
```

Comment:
Probably cleanup is needed here?

Reply:
There is a comment // TO REMOVE before final merge